docs/requirements.docs.in

```diff
@@ -1,6 +1,6 @@
-myst-parser==0.14.0
+myst-parser
 requests
-Sphinx>=2.1,<3
-sphinx-autodoc-typehints==1.10.3
+Sphinx>=4.2.0,<7.0.0
```

Comment:
this is a basic question, but why do we need a version in this range?

Reply:
<4.2.0 gives errors with python 10 and above https://github.com/sphinx-doc/sphinx/issues/9562, and >=7.0.0 gives errors currently https://github.com/kivy/kivy/issues/8230